themes/sandal/theme.config.php

```diff
@@ -1,4 +1,5 @@
 <?php
 return [
     'extends' => 'bootstrap3',
+    'mixins' => ['fontawesome5_icon_mixin'],
```

Comment:
This needs to be removed since we removed the mixin.

Reply:
Damn! I fought so hard to keep this out of my commits!